cpp/include/cugraph_c/sampling_algorithms.h

```diff
@@ -268,6 +268,30 @@ cugraph_type_erased_host_array_view_t* cugraph_sample_result_get_counts(
  */
 void cugraph_sample_result_free(cugraph_sample_result_t* result);
 
+/**
+ * @brief     Create a sampling result (testing API)
+ *
+ * @param [in]   handle         Handle for accessing resources
+ * @param [in]   srcs           Host array view to populate srcs
+ * @param [in]   dsts           Host array view to populate dsts
+ * @param [in]   weights        Host array view to populate weights
+ * @param [in]   counts         Host array view to populate counts
+ * @param [out]  result         Pointer to the location to store the
+ *                              cugraph_sample_result_t*
+ * @param [out]  error          Pointer to an error object storing details of
+ *                              any error.  Will be populated if error code is
+ *                              not CUGRAPH_SUCCESS
+ * @return error code
+ */
+cugraph_error_code_t cugraph_sample_result_create(
```

Comment:
Is the intention to leave this testing API in the final API, or to use this until the actual implementation is available?

If the plan is to leave it in the API permanently, I think we should put the word `test` somewhere in the function name.  Perhaps `cugraph_test_sample_result_create` to indicate that it's a test API.

Reply:
I was thinking it would be kept in the library for unit testing code that handles sample results, without requiring that a sampling algo be called.  I'll rename the function `cugraph_test_sample_result_create` as suggested.